OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -150,39 +150,29 @@ public override Activity Tick(Actor self)
 				return this;
 			}
 
-			var delta = attackAircraft.GetTargetPosition(pos, checkTarget) - pos;
+			var delta = attackAircraft.GetTargetPosition(pos, target) - pos;
```

Comment:
Was replacing `checkTarget` with `target` intentional? This will regress targeting vs units that have disappeared under the fog (`target` refers to the *actual* hidden unit, `checkTarget` to its last known position).

Reply:
Yes, it actually was intentional.

If `useLastVisibleTarget` is `true`, we won't get here anyway (due to the `if` block above), and if it's false, `checkTarget` is equal to `target` anyway.
This was just about consistency (this and some places below were using checkTarget, while other parts below already used target).
I can change everything to `checkTarget` if you think that's better, but like I said, if we got this far in the activity they're equal anyway.

Reply:
Ah, now I see. What you have here is fine.